server/server.js

```diff
@@ -53,20 +55,44 @@ app.use(cors());
 app.use("/build", express.static(path.resolve(__dirname, "../build")));
 
 // route handlers
-app.post('/api',
+app.post(
+  "/api",
   sessionController.isLoggedIn,
   userController.getBoardIds,
   boardController.getBoards,
+
   (_, res) => {
-    res.status(200).json(res.locals.boards)
-  })
+    res.status(200).json(res.locals.boards);
+  }
+);
 
-app.get('/boards/:board_id',
+app.get(
+  "/boards",
+  sessionController.isLoggedIn,
+  boardController.getBoards,
+  (req, res) => {
+    res.status(200).json(res.locals.boards);
+  }
+);
+app.get(
+  "/boards/:board_id",
   sessionController.isLoggedIn,
   boardController.getBoard,
   (req, res) => {
-    res.status(200).json(res.locals.board)
-  })
+    res.status(200).json(res.locals.board);
+  }
+);
+
+// the request body you'll receive is {username: username}
+app.post(
+  "/boards",
+  sessionController.isLoggedIn,
+  boardController.createBoard,
+  userController.addBoardId,
+  (req, res) => {
+    res.status(200).json(res.locals.board);
+  }
```

Comment:
I don't think this needs to be addressed in this PR, though I do agree that we should have a board router asap.

Reply:
Gotcha, makes sense.

Reply:
Agree, that task is something I plan on working on tomorrow.